matminer/featurizers/base.py

```diff
@@ -345,7 +345,7 @@ def featurize_many(self, entries, ignore_errors=False, return_errors=False,
             else:
                 tqdm_func = tqdm.tqdm
             # list() required, tqdm has issues with memory if generator given
-            entries = tqdm_func(list(entries), desc=self.__class__.__name__)
+            # (don't wrap entries in tqdm call) entries = tqdm_func(list(entries), desc=self.__class__.__name__)
```

Comment:
You will still need to wrap the entries in tqdm for the non-multiprocessing case. E.g. when n_jobs is 1.